flybywiresim/a32nx
# refactor(ecam): Convert Pressure page to React

Fixes #[issue_no]

## Summary of Changes


## Screenshots (if necessary)




![ecam-cabin-pressure-page](https://user-images.githubusercontent.com/5834254/135521283-01a3c1a7-55a2-43b4-bb2d-1684a6076630.png)

## References


![CAB-PRESS-1](https://user-images.githubusercontent.com/5834254/131698808-e9a464e3-1e74-4ed6-bbfe-39502d22ec6f.png)





## Additional context



Discord username (if different from GitHub):

## Testing instructions


Probably best to conduct by making a flight
1. Once engines started, turn off each pack in turn: this should be reflected by the pack text and symbol turning amber
2. Adjust the Cabin pressure landing elevation dial: this should be shown by the word AUTO being replaced with MAN and the altitude at the top of the page
3. Get airbourne. You should notice the Outflow valve needle go from vertical to virtual horizontal and an arc should appear
4. Check the Delta PSI, V/S and CAB ALT gauges increase in value as you climb. The gauge should match the value to the right of each gauge
5. Confirm that the values match with the Cruise page (there is sometimes a small descrepency due to rounding).


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
